apps/rush-lib/src/cli/actions/ListAction.ts

```diff
@@ -1,11 +1,15 @@
 // Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
 // See LICENSE in the project root for license information.
 
+// eslint-disable-next-line
+const importLazy = require('import-lazy')(require);
+
 import { BaseRushAction } from './BaseRushAction';
 import { RushCommandLineParser } from '../RushCommandLineParser';
 import { CommandLineFlagParameter } from '@rushstack/ts-command-line';
 import { RushConfigurationProject } from '../../api/RushConfigurationProject';
-import * as Table from 'cli-table';
+// eslint-disable-next-line
+const Table = importLazy('cli-table');
```

Comment:
It'd be really ideal if we did something like this to retain typings information:

```suggestion
import type * as TTable from 'cli-table';
const Table: typeof TTable = importLazy('cli-table');
```